docs/mail.rst

```diff
@@ -0,0 +1,88 @@
+Email basics
+============
+
+The big picture
+---------------
+Sending email involves lots of components.
+
+.. image:: https://res.cloudinary.com/shaka/image/upload/v1542098296/ansible_mailserver/simple_mail_1.png
+
+Components
+----------
+
+Mail Transfer Agent
+^^^^^^^^^^^^^^^^^^^
+Within Internet message handling services (MHS), a message transfer agent or
+mail transfer agent (MTA) or mail relay is software that transfers electronic
```

Comment:
That's a bit heavy.
maybe something like ", a Mail Transfer Agent (MTA, also called Message Tranfer agent, or mail relay) is a software"